FunctionalTableData/CellStyle.swift

```diff
@@ -54,23 +54,27 @@ public struct CellStyle {
 	public var selectionColor: UIColor?
 	/// The view's background color.
 	public var backgroundColor: UIColor?
-	/// The view that is displayed behind the cell’s other content.
+	/// The view that is displayed behind the cell's other content.
+	@available(*, deprecated, message: "Replaced with backgroundViewProvider.")
 	public var backgroundView: UIView?
+	/// Provides the view that is displayed behind the cell's other content.
+	public var backgroundViewProvider: BackgroundViewProvider?
 	/// The tint color to apply to the cell.
 	public var tintColor: UIColor?
 	/// The default spacing to use when laying out content in the view.
 	public var layoutMargins: UIEdgeInsets?
 	/// The radius to use when drawing rounded corners in the view.
 	public var cornerRadius: CGFloat
 	
+	@available(*, deprecated, message: "The `backgroundView` argument is no longer available. Use backgroundViewProvider instead.")
 	public init(topSeparator: Separator.Style? = nil,
 	            bottomSeparator: Separator.Style? = nil,
 	            separatorColor: UIColor? = nil,
 	            highlight: Bool? = nil,
 	            accessoryType: UITableViewCell.AccessoryType = .none,
 	            selectionColor: UIColor? = CellStyle.defaultSelectionColor,
 	            backgroundColor: UIColor? = CellStyle.defaultBackgroundColor,
-	            backgroundView: UIView? = nil,
+	            backgroundView: UIView?,
```

Comment:
Removed the default value here to avoid `Ambiguous use of 'init'`. This also means `CellStyle` initializers without an explicitly set `backgroundView` will default to the new implementation.